cmd/locateCG/locateCG.go

```diff
@@ -0,0 +1,186 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+
+	"github.com/vertgenlab/gonomics/dna"
+	"github.com/vertgenlab/gonomics/exception"
+	"github.com/vertgenlab/gonomics/fasta"
+	"github.com/vertgenlab/gonomics/fileio"
+)
+
+type Settings struct {
+	InFa      string //input fasta file with 2 aligned sequences to compare
+	ChromName string //e.g. chr1
+}
+
+type CGsite struct {
```

Comment:
For a future PR, you can consider making a package for cg, or moving some of the structs and functions to a package like dna!